OpenRA.Game/Graphics/SheetBuilder.cs

```diff
@@ -66,7 +66,7 @@ public Sprite Add(byte[] src, Size size, float2 spriteOffset)
 		{
 			// Don't bother allocating empty sprites
 			if (size.Width == 0 || size.Height == 0)
-				return new Sprite(current, Rectangle.Empty, spriteOffset, channel, BlendMode.Alpha);
+				return new Sprite(current, Rectangle.Empty, spriteOffset, channel, BlendMode.Alpha, 1f);
```

Comment:
alpha = 1 is already the default parameter value, so you don't need to specify it again here.